assets/component-collection-hero.css

```diff
@@ -72,6 +73,15 @@
   padding: 5rem 0 4rem;
 }
 
+.collection-hero__image-container {
+  border: var(--media-border-width) solid rgba(var(--color-foreground), var(--media-border-opacity));
+  border-radius: var(--media-radius);
+  box-shadow: var(--media-shadow-horizontal-offset)
+    var(--media-shadow-vertical-offset)
+    var(--media-shadow-blur-radius)
+    rgba(var(--color-foreground), var(--media-shadow-opacity));
+}
+
```

Comment:
I'm starting to wonder if it would make sense for us to, as a refactor, add some classes that represent each of these things - `media` for things that take media settings, `content-container` for things that take those settings, etc...

**Some benefits**

* Reduction of CSS
* Clearer relationship in the code between the settings and what they affect

The downside would be that some situations do have special cases, but maybe we can make that work anyway.

Something we can think about!

Reply:
I am actually already trying to do this with the media settings in my current PR. I have a global-media-settings class. And we added a `content-containers` class as well.

```
.global-media-settings {
   ... 
}

.global-media-settings:after {
   ...
}

.global-media-settings--full-width,
.content-containers--full-width {
   ... 
}
``` 

I'm going through it now to make sure it applies everywhere well